tools/kagami/src/genesis/generate.rs

```diff
@@ -85,9 +85,29 @@ impl<T: Write> RunArgs<T> for Args {
     }
 }
 
+// This should be consistent with `scripts/test_env.py`
```

Comment:
Can't we modify topology inside `scripts/test_env.py`? 
Because it allows running peer with any number of peers (not only 4).